docs/zh/latest/getting-started.md

```diff
@@ -134,23 +132,23 @@ $ curl "http://127.0.0.1:9080/apisix/admin/services/" -H 'X-API-KEY: edd1c9f0343
 }
 ```
 
-这条路由配置是指：当入站请求满足下述规则时，请求将被转发到 `httpbin.org:80` 上游：
+这条路由配置意味着，所有的入站请求都将被转发到 `httpbin.org:80` 上游，当它们满足下述的 **所有** 规则 (匹配的请求)：
```

Comment:
```suggestion
这条路由配置意味着，当它们满足下述的 **所有** 规则时，所有匹配的入站请求都将被转发到 `httpbin.org:80` 上游，```

Reply:
What do I need to do now？

Reply:
CC @juzhiyuan

Reply:
@gxlct008 Hi, if you agree with my suggestion, just click `Commit suggession`.